doc.md

```diff
@@ -1344,14 +1358,14 @@ GET /api/roles
 <a name='get-role-order'></a>
 #### Retrieve role prioritization order [GET /api/roles/order]
 
-Returns `{ roleIDs }`, where `roleIDs` is an array of role IDs representing the order that roles are applied when [permissions](#permissions) are calculated. Internal roles, such as `_guest` and `_everyone`, are not included.
+Returns `{ roleIDs }`, where `roleIDs` is an array of role IDs representing the order that roles are applied when [permissions](#permissions) are calculated. [Internal roles](spec.md#internal-roles), such as `_user` and `_everyone`, are not included.
 
 <a name='prioritize-roles'></a>
 #### Change role prioritization order [PATCH /api/roles/order]
 
 + requires [permission](#permissions): `manageRoles`
 + `roleIDs` (array of IDs) - The order roles are applied in
-  * Must contain all role IDs just once, except for internal ones such as `_user` and `_everyone`
+  * Must contain all role IDs exactly once (no more, no less), except for [internal roles](spec.md#internal-roles) such as `_user` and `_everyone`
```

Comment:
Not that you've touched this in this PR, but you may want to change this to be clearer that it *should not* contain internal roles. (Currently it sort of sounds like "it may or may not contain internal roles, and may contain it once"; i.e. that the "except" is excepting "exactly once", not "must contain".)